core/src/main/java/io/confluent/connect/storage/StorageSinkConnectorConfig.java

```diff
@@ -89,6 +89,9 @@
       "Filename Offset Zero Pad Width";
 
   public static final String SCHEMA_CACHE_SIZE_CONFIG = "schema.cache.size";
+  // Schema Cache Config Name used by Schema Registry
+  // Ref AvroDataConfig.java in https://github.com/confluentinc/schema-registry
+  public static final String SCHEMAS_CACHE_SIZE_CONFIG = "schemas.cache.config";
```

Comment:
Why can't we use `AvroDataConfig.SCHEMAS_CACHE_SIZE_CONFIG` and avoid hardcoding the value?
Seems to be public